_posts/2023-12-29-czi-funds-pyopensci-2024.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: single
+title: "CZI Funds pyOpenSci"
+excerpt: "CZI has received funding from the Chan Zuckerberg Foundation"
+author: "Leah Wasser"
+permalink: /blog/czi-funds-pyOpenSci-2024.html
+header:
+  overlay_color: "#666"
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+toc: true
+comments: true
+---
+
+## A bright 2024 is ahead for pyOpenSci
+
+We are thrilled to announce that pyOpenSci has received 2 years of funding to cover core operations from the [Chan Zuckerberg Initiative (CZI)](https://chanzuckerberg.com/). These CZI funds will be used to continue critical pyOpenSci work that:
+
+* supports Python open source maintainers in developing the usable and maintainable scientific software the enables open science
+* helps scientists navigate a complex Python packaging ecosystem
+* diversifies the scientific Python open source community
+
+
+## Open science builds trust and enables more people to participate
+
+Open and reproducible science builds trust and accelerates research and
+discovery. Open science supports scientific research that is both transparent and
+reusable. Free and open source software is critical to open science as it ensures that
+the analyses of research data are broadly accessible. To build truly open research workflows, scientists need to use free and open source software (FOSS). FOSS removes the barriers that licenses and other fees may create, particularly for undeserved communities.
+
+### Broad inclusion is critical for both open source and open science
+
+Broad inclusion of individuals in the open source ecosystem which has been found to be [even less diverse then the broader tech community](https://www.wired.com/2017/06/diversity-open-source-even-worse-tech-overall/) is also critical to pyOpenSci's mission. pyOpenSci wants to empower everyone to participate in both the development and use of open source software. This
+empowerment enables open science to reach its full potential.
+
+### Open source maintainers need more support
+Despite the importance of open source software and inclusion to fundamental
```

Comment:
I'm not entirely clear what we're trying to say here. What's throwing me is the phrase "and inclusion"

Reply:
i hear you. I think the rewrite goal was to talk about 
1. how open source is important to open science
2. inclusion is important to both 

maybe there is a way to make that more clear? there are papers about the importance of dei in science and team work in general. but thing link of ideas is a bit tricky

Reply:
ok i implemented a minor fix and just dropped the inclusion as it really didn't make sense there!